web/i18n/eo.yml

```diff
@@ -213,7 +213,7 @@ pages:
         packed: Kompaktigita
         unpacked: Nekompaktigita
         description: Priskribo
-        note: Kelkaj indeksoj ne troviĝas en la datumbazo elŝutebla tie ĉi. La 'kompaktiga' versio ne inkludas tiujn ĉi indeksojn, la 'nekompaktigita' versio inkludas indeksojn, kiujn vi probable bezonos krei post elŝuto.
+        note: Kelkaj indeksoj ne troviĝas en la datumbazo elŝutebla tie ĉi. La ‘kompaktiga’ versio ne inkludas tiujn ĉi indeksojn, la ‘nekompaktigita’ versio inkludas indeksojn, kiujn vi probable bezonos krei post elŝuto.
```

Comment:
I started paying attention to proper punctuation, use apostrophes (') as apostrophes and quotes (‘’) as quotes, I know, that they look almost the same.